bowling.js

```diff
@@ -0,0 +1,53 @@
+// const reducer = (previousValue, currentValue) => previousValue + currentValue;
+
+class Bowling {
+  constructor() {
+    this.rolls = new Array();
```

Comment:
Having the record of rolls a simple array is a nice way to do it, it looks like it allows simple iteration and therefore simple description of strike/spare